### ISSUE TITLE
`:unused-alias` false positive for CLJS string import style

### ISSUE THREAD

**Issue by skynet-gh** [clj-kondo/clj-kondo#2179]
**version**

v2023.09.07

**platform**

linux-amd64 native

**problem**

The new `:unused-alias` linter shows a false positive for CLJS library aliases imported using the `["x" :as x]` format.

**repro**

```sh
$ clj -M:clj-kondo --lang cljs --config '{:linters {:unused-alias {:level :warn}}}' --lint - <<< '(ns foo (:require ["dayjs" :as dayjs])) dayjs'
<stdin>:1:32: warn: Unused alias: dayjs
```

**config**

n/a

**expected behavior**

Expected behavior in this example is to not find any errors since the namespace alias is used.
